docs/core/compatibility/serialization.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Serialization breaking changes
+description: Lists the breaking changes in the serialization category in .NET Core and .NET 5.0 and later.
+ms.date: 07/30/2020
+---
+# Serialization breaking changes
+
+The following breaking changes are documented on this page:
+
+| Breaking change | Introduced version |
+| - | - |
+| [BinaryFormatter.Deserialize rewraps some exceptions in SerializationException](#binaryformatterdeserialize-rewraps-some-exceptions-in-serializationexception) | 5.0 |
+| [JSON serializer exception type changed from JsonException to NotSupportedException](#json-serializer-exception-type-changed-from-jsonexception-to-notsupportedexception) | 3.0 |
```

Comment:
Actually, I think we can delete this breaking change altogether since it was a breaking change from a previous preview version only. Please confirm.

Reply:
Agree. I don't think we doc breaks from preview to preview.